test/framework/src/main/java/org/opensearch/test/OpenSearchSingleNodeTestCase.java

```diff
@@ -211,7 +212,10 @@ protected final Collection<Class<? extends Plugin>> pluginList(Class<? extends P
 
     /** Additional settings to add when creating the node. Also allows overriding the default settings. */
     protected Settings nodeSettings() {
-        return Settings.EMPTY;
+        // By default, for tests we will put the target slice count of 2. This will increase the probability of having multiple slices
```

Comment:
Do you think there is still value in testing the concurrent segment search case with 1 slice?

Reply:
was thinking that code path is mostly same between 1 or more slices like reducing and creating the final results. So it is more a subset of the slice > 1 case. Also the max slice of 2 will still have coverage for single slice if the number of segments created by the tests is 1.